keycloak/keycloak-admin-ui
# Release scripts

## Brief Description
This PR contains our "Keycloak standard" scripts needed to do a release.  They are based on the scripts used in other Keycloak projects.

## Verification Steps
I was only able to verify that `get-version.sh` and `set-version.sh` are working properly.  `release.sh` requires our org.keycloak credentials to the maven central repo, so I think only Stian can run it.

As a Windows user, I don't write a lot of shell scripts.  So please review and let me know if you see anything obviously wrong, especially with `release.sh`.
